data/names/ru.json

```diff
@@ -3447,31 +3535,50 @@
       "Винсон",
       "Винтер",
       "Витале",
-      "Виттен",
       "Витт",
+      "Виттен",
       "Вишневски",
+      "Вишневский",
+      "Вмятина",
+      "Во",
       "Воган",
+      "Волк",
       "Вольф",
-      "Вонг",
+      "Воля",
+      "Вомак",
       "Вон",
+      "Вонг",
+      "Ворота",
```

Comment:
This is a translation of Gates, the last name, as a common noun. It would be transcribed as Гейтс, as in [Билл Гейтс](https://ru.wikipedia.org/wiki/%D0%93%D0%B5%D0%B9%D1%82%D1%81,_%D0%91%D0%B8%D0%BB%D0%BB), the former CEO of Microsoft.